testing/regress/ecl/dfusupercopy.ecl

```diff
@@ -0,0 +1,51 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2024 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//noroxie
+//nohthor
+
+import Std.System.Thorlib;
+import Std.File AS FileServices;
+import $.setup;
+prefix := setup.Files(false, false).QueryFilePrefix;
+
+// Generate Data
+layout_user := RECORD
+   STRING20 user;
+END;
+
+ds1 := DATASET([{'Ned'},{'Robert'}, {'Jaime'}, {'Catelyn'}, {'Cersei'}, {'Daenerys'}, {'Jon'}], layout_user, DISTRIBUTED );
+ds2 := DATASET([{'Sansa'}, {'Arya'}, {'Robb'}, {'Theon'}, {'Bran'}, {'Joffrey'}, {'Hound'}, {'Tyrion'}], layout_user, DISTRIBUTED);
+ds3 := DATASET([{'Arya'}, {'Robb'}, {'Theon'}, {'Bran'}, {'Joffrey'}, {'Hound'}, {'Tyrion'}], layout_user, DISTRIBUTED);
+
+SEQUENTIAL(
+    OUTPUT(ds1, , prefix + 'subdata1', OVERWRITE),
+    OUTPUT(ds2, , prefix + 'subdata2', OVERWRITE),
+    OUTPUT(ds3, , prefix + 'subdata3', OVERWRITE),
+    FileServices.CreateSuperFile(prefix + 'superdata'),
+    FileServices.StartSuperFileTransaction(),
+    FileServices.AddSuperFile(prefix + 'superdata', prefix + 'subdata1'),
+    FileServices.AddSuperFile(prefix + 'superdata', prefix + 'subdata2'),
+    FileServices.AddSuperFile(prefix + 'superdata', prefix + 'subdata3'),
+    FileServices.FinishSuperFileTransaction(),
+    FileServices.Copy(sourceLogicalName := prefix + 'superdata', destinationGroup := Thorlib.group(), destinationLogicalName := prefix + 'super_copy', ALLOWOVERWRITE := true),
```

Comment:
>destinationGroup := Thorlib.group()

is there any reason to define this at all?
thorlib.group() in k8s doesn't make sense - destinationPlane should be used..

But, neither needs to be defined afaics, defaults would be good.

Reply:
ok, it happens to work (in k8s) because Thorlib.group() makes no sense in k8s context, so it returns "". That in turn is treated by File.Copy to mean none provided, which then make it default to use the plane/group of the source file being copied.
So passing "" would be more correct here. Can you change to that and check it works?
Assuming it does, it would be sensible to allow this parameter to be optional, omit it, and let it perform the default behaviour if not supplied - which is what we want here.

However, file service calls like this (e.g. Spray too) should really be able to explicitly use a destinationPlane.
I've opened a separate JIRA: https://hpccsystems.atlassian.net/browse/HPCC-32844